pkg/helm/option.go

```diff
@@ -492,9 +492,9 @@ func WithMaxHistory(max int32) HistoryOption {
 }
 
 // NewContext creates a versioned context.
-func NewContext() context.Context {
+func NewContext(ctx context.Context) context.Context {
```

Comment:
As a note, we generally avoid introducing breaking changes like this one. What if we introduced a new function called `FromContext` that accepted a `context.Context`? That way `NewContext()` can stay as-is and internally it's just a `return FromContext(context.TODO())`.

That would also fix the backwards compatibility breakers that changed the contexts from a `.TODO()` to a `.Background()`. in client.go

Reply:
Done